x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/index.tsx

```diff
@@ -57,29 +57,27 @@ const ShiftedEuiText = styled(EuiText)`
 
 const RUM_PAGE_LOAD_TYPE = 'page-load';
```

Comment:
Have you not rebased yet, or should this go into https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/apm/common/transaction_types.ts#L9 ?

Reply:
Fixed in e865ebe1b5.